### PR TITLE
Update getting started (and some other changes)

### PR DESCRIPTION
**Pull request by bramkragten**
## Proposed change

Updates the getting started page to explain the different installation methods better.

Some general style changes to align everything better (border radius, borders, shadows)
Fixes in energy landing page

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [X] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [ ] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/installation/index.html [hunk 1]
```diff
@@ -6,263 +6,559 @@
...
-                        <li>Home Assistant Operating System</li>
-                    </ul>
-                </div>
-            </div>
-            {% include assets/chevron_right.html %}
+  <span class="label easiest">Easiest</span>
+
+  <h2>Plug and play with Home Assistant Green</h2>
+  <p>
+    The affordable Home Assistant Green is the easiest way you can start using
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
    The affordable Home Assistant Green is the easiest way to start using
```

File: source/installation/index.html [hunk 2]
```diff
@@ -6,263 +6,559 @@
...
+              <ul>
+                <li>Assembling a Raspberry Pi setup</li>
+                <li>Flashing a Raspberry Pi</li>
+              </ul>
             </div>
-            {% include assets/chevron_right.html %}
+            <div>
+              <b>TOOLS REQUIRED</b>
+              <ul>
+                <li>Raspberry Pi 3 / 4 / or newer</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
                <li>Raspberry Pi 3 or 4 with power supply</li>
```
removing newer, making a note to add it back in as soon as Pi 5 is supported

File: source/installation/index.html [hunk 3]
```diff
@@ -6,263 +6,559 @@
...
-                        <li>Home Assistant Container</li>
-                        <li>Home Assistant Core</li>
-                    </ul>
-                </div>
+  <h2>Home Assistant Operating System</h2>
+  <p>
+    Home Assistant can repurpose and be installed on all sorts of hardware, such
+    as ODroid and generic x86-64 machines. Home Assistant Operating System
+    allows you to install Home Assistant on these devices even if you have
+    little to no Linux experience.
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
    Home Assistant can be repurposed and installed on various hardware, such
    as an Odroid or a generic x86-64 machine. The Home Assistant Operating System
    allows you to install Home Assistant on these devices even if you have
    little to no Linux experience.
```

File: source/installation/index.html [hunk 4]
```diff
@@ -6,263 +6,559 @@
...
-                    </ul>
-                </div>
+        <div class="content">
+          <h3>Home Assistant Green</h3>
+          <p>The easiest way to get started with Home Assistant</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Basic smart home setup skills</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
                <li>Interest in setting up a smart home</li>
```

File: source/installation/index.html [hunk 5]
```diff
@@ -6,263 +6,559 @@
...
+        </div>
+        <div class="content">
+          <h3>Home Assistant Yellow</h3>
+          <p>The powerful way to run Home Assistant</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Very basic electronics skills akin to building a PC</li>
+                <li>Flashing a Raspberry Pi</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
                <li>You're comfortable doing the following tasks based on instructions:</li>
                <li>Installing a compute module and a heat sink</li>
                <li>Flashing a Raspberry Pi</li>
```

File: source/installation/index.html [hunk 6]
```diff
@@ -6,263 +6,559 @@
...
+  </p>
+
+  <div class="installations-card">
+    <div class="material-card text">
+      <div class="content-container">
+        <div class="image">
+          <img src="/images/installation/odroid.jpg" />
+        </div>
+        <div class="content">
+          <h3>Install Home Assistant on ODroid devices</h3>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
          <h3>Install Home Assistant on Odroid devices</h3>
```

File: source/installation/index.html [hunk 7]
```diff
@@ -6,263 +6,559 @@
...
+          <p>A more powerful alternative to Raspberry Pi</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Understanding of hardware requirements</li>
+                <li>Writing boot images to your boot medium</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
                <li>You're comfortable doing the following tasks based on instructions:</li>
                <li>Writing boot images to your boot medium</li>
                <li>Installing an SD card or eMMC</li>
```

File: source/installation/index.html [hunk 8]
```diff
@@ -6,263 +6,559 @@
...
+              </ul>
+            </div>
+            <div>
+              <b>TOOLS REQUIRED</b>
+              <ul>
+                <li>An Odroid device</li>
+                <li>MicroSD card</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
                <li>MicroSD card or eMMC</li>
```

File: source/installation/index.html [hunk 9]
```diff
@@ -6,263 +6,559 @@
...
+        </div>
+        <div class="content">
+          <h3>Install Home Assistant on x86-64 machines</h3>
+          <p>Repurpose workstation hardware to run Home Assistant</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Building a PC or formatting a PC from scratch</li>
+                <li>Writing boot images to your boot medium</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
                <li>You can use a command line and install a boot medium on your hardware</li>
                <li>You're comfortable configuring the BIOS based on instructions.</li>
```

File: source/installation/index.html [hunk 10]
```diff
@@ -6,263 +6,559 @@
...
+    <li>
+      <b>Home Assistant Core:</b> Manual installation using Python virtual
+      environment.
+    </li>
+  </ul>
+  <p>
+    Please note that these installation methods, while they provide some
+    columnsibility for advanced users, they may also have some major limitations
+    on usage of Add-ons and other important Home Assistant features. Compare
+    installation methods here.
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
    Note that while these installation methods may provide some
    features for advanced users, they may also have some major limitations.
    For example, add-ons and other important Home Assistant features may not be available. Compare
    installation methods here.
```

**Review by c0ffeeca7 — COMMENTED** [review]
minor suggestions. Not sure the text fits the space.

File: source/installation/index.html [hunk 11]
```diff
@@ -6,263 +6,561 @@
...
+          <img src="/images/installation/yellow.jpg" />
+        </div>
+        <div class="content">
+          <h3>Home Assistant Yellow</h3>
+          <p>The powerful way to run Home Assistant</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>You're comfortable doing the following tasks based on instructions:</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
                <li>You're comfortable following instructions on:</li>
```

File: source/installation/index.html [hunk 12]
```diff
@@ -6,263 +6,561 @@
...
+          <img src="/images/installation/odroid.jpg" />
+        </div>
+        <div class="content">
+          <h3>Install Home Assistant on Odroid devices</h3>
+          <p>A more powerful alternative to Raspberry Pi</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>You're comfortable doing the following tasks based on instructions:</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
                <li>You're comfortable following instructions on:</li>
```

File: source/installation/index.html [hunk 13]
```diff
@@ -6,263 +6,561 @@
...
+                <li>Writing boot images to your boot medium</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
                <li>Writing boot images</li>
```

File: source/installation/index.html [hunk 14]
```diff
@@ -6,263 +6,561 @@
...
-                    </ul>
-                </div>
+        <div class="content">
+          <h3>Home Assistant Green</h3>
+          <p>The easiest way to get started with Home Assistant</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Interest in setting up a smart home</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html:40]:
```suggestion
                <li>Interest in setting up a smart home</li>
                <li>Know how to find an IP address on your router</li>
```

File: source/installation/index.html [hunk 15]
```diff
@@ -6,263 +6,561 @@
...
+        </div>
+        <div class="content">
+          <h3>Install Home Assistant on Raspberry Pi</h3>
+          <p>A low-cost DIY solution to get started with Home Assistant</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Assembling a Raspberry Pi setup</li>
+                <li>Flashing a Raspberry Pi</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html:92]:
```suggestion
                <li>Flashing a Raspberry Pi</li>
                <li>Know how to find an IP address on your router</li>
```

File: source/installation/index.html [hunk 16]
```diff
@@ -6,263 +6,561 @@
...
+          <img src="/images/installation/yellow.jpg" />
+        </div>
+        <div class="content">
+          <h3>Home Assistant Yellow</h3>
+          <p>The powerful way to run Home Assistant</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>You're comfortable following instructions on:</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html:139]:
```suggestion
                <li>Know how to find an IP address on your router</li>
                <li>You're comfortable following instructions on:</li>
```

File: source/installation/index.html [hunk 17]
```diff
@@ -6,263 +6,561 @@
...
+          <img src="/images/installation/odroid.jpg" />
+        </div>
+        <div class="content">
+          <h3>Install Home Assistant on Odroid devices</h3>
+          <p>A more powerful alternative to Raspberry Pi</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>You're comfortable following instructions on:</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html:194]:
```suggestion
                <li>Know how to find an IP address on your router</li>
                <li>You're comfortable following instructions on:</li>
```

File: source/installation/index.html [hunk 18]
```diff
@@ -6,263 +6,561 @@
...
+          </svg>
+        </div>
+        <div class="content">
+          <h3>Install Home Assistant on Windows</h3>
+          <p>Use Home Assistant OS on a VM, or Core on WSL</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Advanced knowledge of Windows</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html:368]:
```suggestion
                 <li>Know how to find an IP address on your router</li>
                <li>Advanced knowledge of Windows</li>
```

File: source/installation/index.html [hunk 19]
```diff
@@ -6,263 +6,561 @@
...
+          </svg>
+        </div>
+        <div class="content">
+          <h3>Install Home Assistant on other systems</h3>
+          <p>Use Home Assistant on virtual machines, NAS, and more</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Advanced knowledge of Windows</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html:404]:
```suggestion
                <li>Know how to find an IP address on your router</li>
                <li>Advanced knowledge of Windows</li>
```

**Review by c0ffeeca7 — COMMENTED** [review]
add prereq that you need to know how to find an IP address on your router. Input by cogneato.

File: source/installation/index.html [hunk 20]
```diff
@@ -6,263 +6,567 @@
...
-                </div>
+        <div class="content">
+          <h3>Home Assistant Green</h3>
+          <p>The easiest way to get started with Home Assistant</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Interest in setting up a smart home</li>
+                <li>Know how to find an IP address on your router</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
```

File: source/installation/index.html [hunk 21]
```diff
@@ -6,263 +6,567 @@
...
+        <div class="content">
+          <h3>Install Home Assistant on Raspberry Pi</h3>
+          <p>A low-cost DIY solution to get started with Home Assistant</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Assembling a Raspberry Pi setup</li>
+                <li>Flashing a Raspberry Pi</li>
+                <li>Know how to find an IP address on your router</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
```

File: source/installation/index.html [hunk 22]
```diff
@@ -6,263 +6,567 @@
...
+          <img src="/images/installation/yellow.jpg" />
+        </div>
+        <div class="content">
+          <h3>Home Assistant Yellow</h3>
+          <p>The powerful way to run Home Assistant</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Know how to find an IP address on your router</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
```

File: source/installation/index.html [hunk 23]
```diff
@@ -6,263 +6,567 @@
...
+          <img src="/images/installation/odroid.jpg" />
+        </div>
+        <div class="content">
+          <h3>Install Home Assistant on Odroid devices</h3>
+          <p>A more powerful alternative to Raspberry Pi</p>
+          <div class="columns">
+            <div>
+              <b>SKILLS REQUIRED</b>
+              <ul>
+                <li>Know how to find an IP address on your router</li>
```
Comment (c0ffeeca7) [inline source/installation/index.html]:
```suggestion
```

**Review by c0ffeeca7 — DISMISSED** [review]

**Review by c0ffeeca7 — DISMISSED** [review]

**Review by frenck — APPROVED** [review]
Let's go!

Thanks, @bramkragten & @c0ffeeca7!

../Frenck

**Pull request merged**